[I 140514 00:07:25 application:38] start server...
[E 140514 00:07:38 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'If-None-Match': '"502c9457348f887d4834603b179c1d80a6fe4917"'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 19, in get
        loging.info("entering IndexHandler...get")
    NameError: global name 'loging' is not defined
[E 140514 00:07:38 web:1728] 500 GET / (::1) 2.17ms
[I 140514 00:07:59 autoreload:191] /home/liqing/Dropbox/uestc/crawler_uestc/exhibition/routing.py modified; restarting server
[I 140514 00:07:59 application:38] start server...
[I 140514 00:08:18 autoreload:191] /home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py modified; restarting server
[I 140514 00:08:19 application:38] start server...
[I 140514 00:08:24 index:19] entering IndexHandler...get
[E 140514 00:08:24 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 615, in render
        html = self.render_string(template_name, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:08:24 web:1728] 500 GET / (::1) 82.24ms
[I 140514 00:09:32 autoreload:191] /home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py modified; restarting server
[I 140514 00:09:32 application:38] start server...
[I 140514 00:09:38 index:19] entering IndexHandler...get
[E 140514 00:09:38 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render_string("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:09:38 web:1728] 500 GET / (::1) 4.61ms
[I 140514 00:10:15 autoreload:191] /home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py modified; restarting server
[I 140514 00:10:15 application:38] start server...
[I 140514 00:10:34 index:19] entering IndexHandler...get
[E 140514 00:10:34 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render_string("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:10:34 web:1728] 500 GET / (::1) 3.95ms
[I 140514 00:11:37 index:19] entering IndexHandler...get
[E 140514 00:11:37 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render_string("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:11:37 web:1728] 500 GET / (::1) 4.47ms
[I 140514 00:11:40 index:19] entering IndexHandler...get
[E 140514 00:11:40 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render_string("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:11:40 web:1728] 500 GET / (::1) 2.10ms
[I 140514 00:11:44 index:19] entering IndexHandler...get
[E 140514 00:11:44 web:1305] Uncaught exception GET / (::1)
    HTTPRequest(protocol='http', host='localhost:8000', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36', 'Connection': 'keep-alive', 'Cookie': '_xsrf=a2ea9d0b78aa4fbdb08a5eb61e1100d6', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete
        callback()
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py", line 20, in get
        self.render_string("<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>")
      File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 719, in render_string
        t = loader.load(template_name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 343, in load
        self.templates[name] = self._create_template(name)
      File "/usr/local/lib/python2.7/dist-packages/tornado/template.py", line 370, in _create_template
        f = open(path, "rb")
    IOError: [Errno 2] No such file or directory: '/home/liqing/Dropbox/uestc/crawler_uestc/exhibition/template/<html><title>uestc_show</title><body>it works! but have nothing now!</body></html>'
[E 140514 00:11:44 web:1728] 500 GET / (::1) 2.17ms
[I 140514 00:12:11 autoreload:191] /home/liqing/Dropbox/uestc/crawler_uestc/exhibition/handler/index.py modified; restarting server
[I 140514 00:12:11 application:38] start server...
[I 140514 00:12:14 index:19] entering IndexHandler...get
[I 140514 00:12:14 index:21] leaving IndexHandler...get
[I 140514 00:12:14 web:1728] 200 GET / (::1) 11.74ms
[I 140514 00:12:17 index:19] entering IndexHandler...get
[I 140514 00:12:17 index:21] leaving IndexHandler...get
[I 140514 00:12:17 web:1728] 304 GET / (::1) 1.82ms
